bin/node-template/scripts/docker_run.sh

```diff
@@ -2,9 +2,72 @@
 # This script is meant to be run on Unix/Linux based systems
 set -e
 
-echo "*** Start Substrate node template ***"
-
+printf "*** Start Substrate node template ***\n"
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
+PARENT_DIR=$(dirname "$SCRIPT_DIR")
+mkdir -p "$PARENT_DIR/.local"
+mkdir -p "$PARENT_DIR/.cargo/registry"
```

Comment:
Could you maybe put them into the `target/` dir, so I wont forget deleting them eventually?  
Also do you know if there is a way to not make docker chown these folders with root?

Reply:
cheers, i'm working on fixing all that, i'll push some commits and tell you when its done